.bit/responses/3.3-Week 3 Step 3.md

```diff
@@ -39,17 +39,18 @@ Now we'll want to handle cases where the POST request's body happens to be empty
 <details>
 <summary>:question: How do I catch empty POST requests?</summary>
 
-Use an if-else statement to catch when `body == null`. If it's empty, set the `responseMessage` to "Sorry! No image attached." Otherwise, you can safely parse the body!
+Use an try-catch statement to catch when `parse-multipart` is unable to parse the empty body. If catches an error, set the `responseMessage` to "Sorry! No image attached." Otherwise, you can safely parse the body!
 
 ```js
 var responseMessage = ""
-if (body == null) {
-    responseMessage = "Sorry! No image attached."
-} else {
+try {
     var password = // get the header called "codename"
     // use parse-multipart to parse the body
     // determine the file-type here!
     responseMessage = await uploadFile(parsedBody, ext, ?);
```

Comment:
maybe mention that they should replace what goes in the question mark, because they might just try to just pass in the question mark. Also mention that they will need to add a parameter to the `uploadFile` function for the codename